tensorboard/components/tf_data_selector/tf-data-select-row.ts

```diff
@@ -130,6 +130,8 @@ Polymer({
       const fetchRuns = requestManager.request(tf_backend.getRouter().runs());
       return Promise.all([fetchRuns]).then(([runs]) => {
         this.set('_runs', Array.from(new Set(runs)).map(runName => ({
+          // /data/runs endpoint does not return ids. In case of logdir data
+          // source, the ids are missing.
           id: runName,
```

Comment:
Could this be set to null instead of the runName?  The ID is supposed to be a number anyway so using runName as a dummy value seems not quite right.

Reply:
Done! Thanks for catching that